src/components/dev-hub/podcast-player/player-contols.js

```diff
@@ -0,0 +1,98 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+import Button from '~components/dev-hub/button';
+import FastForwardIcon from '~components/dev-hub/icons/audio-player/fast-forward-icon';
+import PauseIcon from '~components/dev-hub/icons/audio-player/pause-icon';
+import PlayIcon from '~components/dev-hub/icons/audio-player/play-icon';
+import RewindIcon from '~components/dev-hub/icons/audio-player/rewind-icon';
```

Comment:
A nice cleanup piece could be to have an `index.js` file in `~components/dev-hub/icons/audio-player` which we import all icons from